### PR TITLE
Simple tests

### PR DESCRIPTION
**Pull request by Faryshta**
| Q | A |
| --- | --- |
| Is bugfix? | no |
| New feature? | no |
| Breaks BC? | no |
| Tests pass? | yes |

Simplifies how the `migrate/create` command tests are run mostly by separating each generated file into their own file.

### CODE REVIEW

**Comment by SilverFire** [comment]
Looks good. I'd move those files from `migrate_create` under the `tests/data/console` directory in order to separate data and actual tests. What do you think about it?

File: tests/framework/console/controllers/MigrateControllerTestTrait.php [hunk 1]
```diff
@@ -42,6 +42,25 @@ public function tearDownMigrationPath()
         FileHelper::removeDirectory($this->migrationPath);
     }
 
+    public function assertFileContent($expectedFile, $class)
+    {
+        $this->assertEqualsWithoutLE(
+            include __DIR__ . "/migrate_create/$expectedFile.php",
```
Comment (SilverFire) [inline tests/framework/console/controllers/MigrateControllerTestTrait.php]: I'd prefer `Yii::getAlias("@yiiunit/data/console/migrate_create/$expectedFile.php")`

Comment (Faryshta) [inline tests/framework/console/controllers/MigrateControllerTestTrait.php]: ok will do.

**Comment by Faryshta** [comment]
@SilverFire done

**Comment by SilverFire** [comment]
Thank you, @Faryshta

**Pull request merged**